SkyBlock/addons/lootbox.sk

```diff
@@ -430,6 +437,8 @@ function startlootbox(p:player,tier:integer):
         set {_crates::%loop-index%} to {_crates::%loop-index%} parsed as integer
         #
         # > Remove one from the current tier of the stored lootboxes.
+        if {_crates::%loop-index%} < 1:
+          stop
```

Comment:
By stopping here, the user can not just move the item out of the selected tool slot and get endless lootboxes, once it reaches a value that is smaller than 1, it will prevent it from continue.